### DISCUSSION TITLE
How to stop getting "Path to Haxe executable is not valid: 'haxe.exe'..." in VSCode

### DISCUSSION THREAD

**Discussion by acuaroo** [HaxeFlixel/flixel#2369] [Q&A] [answered]
# Hello!
I just started HaxeFlixel today, and was following a tutorial. I decided to wrap up, and leave the project be, but when I returned to it in VSCode, I was greeted with:

![image](https://user-images.githubusercontent.com/67112172/128091902-ce185385-2e18-4b2d-87fc-bf251ce4c563.png)

I looked it up and the issue was for multi-root workspaces, but I closed all workspaces & tried to open up the folder & had the same issue. I am wondering if I didn't stop the multi-root workspace, or it is something else.

I've tried searching online but the 2 things I could find lead to multi-root workspaces. Not sure how to reproduce this, but I created a project, saved it & closed VSCode, then once I tried opening it later, it gave this message.

## Edit: Lime is already installed, if I try to click the install button it says it failed

**Comment by acuaroo** [comment, marked as answer]
Hm, I re-installed Haxe and it seems to work all of a sudden...

**Reply by NoodleSushi** [reply]
I had to re-install Haxe every time I switch my computer on, any advice/workarounds?
